src/components/index.ts

```diff
@@ -32,6 +32,11 @@ export type {BannerProps, BannerStatus} from './Banner';
 export {Breadcrumbs} from './Breadcrumbs';
 export type {BreadcrumbsProps} from './Breadcrumbs';
 
+// eslint-disable-next-line @typescript-eslint/camelcase
+export {BulkActions as _SECRET_INTERNAL_BulkActions} from './BulkActions';
+// eslint-disable-next-line @typescript-eslint/camelcase
+export type {BulkActionsProps as _SECRET_INTERNAL_BulkActionsProps} from './BulkActions';
```

Comment:
How do you feel about this? We've used `_SECRET_INTERNAL_` in the past. `Unstable` is also an option.

Reply:
I think I prefer unstable, unless this will remain unstable forever. If that's what you're thinking then maybe something that labels this export as for use within the admin / apps only would make more sense